core/templates/domain/topics_and_skills_dashboard/TopicsAndSkillsDashboardFilterObjectFactory.ts

```diff
@@ -43,8 +48,8 @@ export class TopicsAndSkillsDashboardFilter {
   reset(): void {
     this.category = '';
     this.keyword = '';
-    this.sort = '';
-    this.status = '';
+    this.sort = null;
```

Comment:
One question about this. What happens when there is no selection made by the user? Is the field blank or is there a default selection? It seems like it ought to be the latter, so that it is always valid.

In general, for "type in" fields, an empty string is fine, but for "select from a dropdown" fields, have the default option always be an option in the dropdown.

Reply:
Ok noted for future work. Done.
These are how they look now.
![image](https://user-images.githubusercontent.com/30312043/82984071-04944180-a00f-11ea-8762-91c141c56b4b.png)